apps/proxy/src/proxy/proxy_test.py

```diff
@@ -91,7 +91,7 @@ def finish():
     # Make sure the server thread is done.
     print("Closing thread " + str(thread))
     thread.join(10.0) # Wait at most 10 seconds
-    assert_false(thread.isAlive())
+    assert_false(thread.is_alive())
```

Comment:
Do we not need to check for old python versions (py2.7?) anymore?

Reply:
We are testing for python2.7 as well. As I mentioned in the description `is_alive() got added in python2.6 so no problem for python2.7 and python3.8 based Hue`